### ISSUE TITLE
Set Up PYPI_TOKEN for the Release on PyPI

### ISSUE THREAD

**Issue by ahnaf-tahmid-chowdhury** [sigma-toolkit/moab-python#2]
I think, we are ready to publish the first release of MOAB 5.5.1 on PyPI. However, I do not have sufficient permissions to complete the process by myself. I would like to kindly request @vijaysm to set a private environment variable named `PYPI_TOKEN` and add the token from the PyPI site. Once the token is configured, we are ready to run the workflow and proceed with the release.

**Comment by vijaysm** [comment]
I think we should wait until MOAB 5.6 to push the release. 5.5.1 does not have any of your changes and we are getting ready to spin up a candidate soon. Few pending items remain and we should be good to go when those are done.

**Comment by vijaysm** [comment]
And yes, I can take care of adding the PYPI token and getting the setup running for you to test.

**Comment by shimwell** [comment]
just to mention PYPI tokens are no longer needed, trusted publishing is now possible
https://docs.pypi.org/trusted-publishers/

**Comment by ahnaf-tahmid-chowdhury** [comment]
Both way we need to setup PyPI for moab.

**Comment by ahnaf-tahmid-chowdhury** [comment]
The workflow has been update, it no longer requires setting up a 
`PYPI_TOKEN`, but still necessitates PyPi account configuration with moab.

https://github.com/sigma-toolkit/moab-python/actions/runs/14055488487

**Comment by vijaysm** [comment]
@ahnaf-tahmid-chowdhury let me know if you still need me to set up something to get this integration working.

**Comment by shimwell** [comment]
I would recommend setting up trusted publishing on pypi

once you login to this url then enter this information

https://pypi.org/manage/account/publishing/

![Image](https://github.com/user-attachments/assets/09295c7e-8492-47dc-9067-33efb22e1c07)

**Comment by ahnaf-tahmid-chowdhury** [comment]
Hi @vijaysm, are there any PRs you'd like me to review that are holding up the next release?

**Comment by vijaysm** [comment]
Not at the moment. We are trying to get some pending changes merged for our climate project. Once that is in place, will tag the release. Thanks for checking. A release is long overdue.

**Comment by ahnaf-tahmid-chowdhury** [comment]
I think this is a good time to make a release here as well so we can generate a PyPI wheel.

**Comment by vijaysm** [comment]
Agreed! Let me know if you need anything from me. I'm on vacation from tomorrow until next Monday, however. Will have my phone - so can review if needed.

**Comment by ahnaf-tahmid-chowdhury** [comment]
If you’ve already set up trusted publishing on PyPI as suggested by @shimwell, then the only thing left to do is create a release tag here with version 5.6.0.

**Comment by vijaysm** [comment]
I ran into issues when setting up trusted publishing. 

<img width="468" height="853" alt="Image" src="https://github.com/user-attachments/assets/2c2d6cdb-cdcb-4cb9-9219-64f0ee1d4791" />

**Comment by vijaysm** [comment]
Trying https://pypi.org/project/moab/ leads to a 404 error. So the name is still available. Thoughts?

I'll be on mobile for the rest of the week, but I can still set this up once we resolve the above error.

**Comment by shimwell** [comment]
Perhaps this is the pypi ultranormalization name rule which prevents names similar to existing projects

**Comment by ahnaf-tahmid-chowdhury** [comment]
What about pymoab?

**Comment by vijaysm** [comment]
Using pymoab worked. Perhaps there is a minimum length for the project name? Anyway, this project is now pending publishers.

**Comment by ahnaf-tahmid-chowdhury** [comment]
If we decide to use the project name pymoab, we should also update the pyproject.toml file. Maybe we should create a new release, version 5.6.1?

**Comment by ahnaf-tahmid-chowdhury** [comment]
Once it is done we can create  a release tag in this repo to run the CI which may automatically configure the publisher.

**Comment by shimwell** [comment]
does [this line](https://bitbucket.org/fathomteam/moab/src/3fa8ae9e8025e0c1349792dee1f39b82ad8c1a33/pyproject.toml#lines-11) in the pyproject.toml need changing to ```name=pymoab```
